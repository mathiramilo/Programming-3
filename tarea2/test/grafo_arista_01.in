leer_mst
3 no-dirigido
1: 2 p5 3 p3 .
2: .
3: .
0: .
grafo_arista 2 3 1
Fin