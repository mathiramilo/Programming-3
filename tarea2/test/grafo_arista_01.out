1>2>Imprimiendo Grafo
Cantidad de vÃ©rtices: 3
Cantidad de aristas: 2
1: (3,3).
2: (3,1).
3: (1,3)(2,1).
3>Fin.
